dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/angular-topology-editor.js

```diff
@@ -118,11 +118,16 @@
                         scope.canvas = Canvas();
                         scope.canvas.init(uniqid, canvasOptions);
                         Status.init(scope.canvas);
-
-                        scope.$watch('topology', function (newValue) {
-                            Status.fromjson(scope.topology);
-                            scope.canvas.restart();
-                        });
+								Status.fromjson(scope.topology);
```

Comment:
@rsucasas : Status.fromjson should be inside the $watch, and executed only when seaCloudsApplicationId changes.